### PR TITLE
pod_coverage/update_increased_naked_subs_message

### PR DESCRIPTION
**Pull request by ramin-fs**
- https://redmine.deriv.cloud/issues/38461#pod_coverage/update_increased_naked_subs_message

### CODE REVIEW

File: lib/Test/Pod/CoverageChange.pm [hunk 1]
```diff
@@ -254,7 +254,8 @@ Change the %s => %s in the %s file please.", $package, $package, $naked_subs_cou
                 ));
             next;
         } elsif (!$fully_covered && $naked_subs_count > $max_expected_naked_subs) {
-            $Test_Builder->ok(0, sprintf('Your last changes increased the number of naked subs in the %s package.', $package));
+            $Test_Builder->ok(0, sprintf("Your last changes increased the number of naked subs in the %s package from %s to %s. Please add sube for your new subs.",
```
Comment (sarah-deriv) [inline lib/Test/Pod/CoverageChange.pm]: typo "sube" ???

File: t/multiple_module_test.t [hunk 2]
```diff
@@ -21,14 +21,14 @@ subtest 'every modules will check in the process' => sub {
     test_out(
         "not ok 1 # TODO & SKIP You have 0.00% POD coverage for the module 't::Nopod'.",
         "not ok 2 # TODO & SKIP You have 33.33% POD coverage for the module 't::PartiallyCoveredPod'.",
-        "not ok 3 - Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package.",
+        "not ok 3 - Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package from 1 to 2. Please add sube for your new subs.",
```
Comment (sarah-deriv) [inline t/multiple_module_test.t]: same here

File: lib/Test/Pod/CoverageChange.pm (hunk 1 above)
Comment (shantanu-deriv) [inline lib/Test/Pod/CoverageChange.pm]:
```suggestion
            $Test_Builder->ok(0, sprintf("Your last changes increased the number of naked subs in the %s package from %s to %s. Please add pod for your new subs.",
```

File: t/multiple_module_test.t (hunk 2 above)
Comment (shantanu-deriv) [inline t/multiple_module_test.t]:
```suggestion
        "not ok 3 - Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package from 1 to 2. Please add pod for your new subs.",
```

File: t/multiple_module_test.t [hunk 3]
```diff
@@ -21,14 +21,14 @@ subtest 'every modules will check in the process' => sub {
...
         "ok 4 - Pod coverage on t::CorrectPod",
         "ok 5 - Pod structure is OK in the file t/CorrectPod.pm.",
         "not ok 6 # TODO & SKIP There is no POD in the file t/Nopod.pm.",
         "ok 7 - Pod structure is OK in the file t/PartiallyCoveredPod.pm.",
     );
     test_diag(
-        "  Failed test 'Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package.'",
+        "  Failed test 'Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package from 1 to 2. Please add sube for your new subs.'",
```
Comment (shantanu-deriv) [inline t/multiple_module_test.t]:
```suggestion
        "  Failed test 'Your last changes increased the number of naked subs in the t::PartiallyCoveredPod package from 1 to 2. Please add pod for your new subs.'",
```

File: t/no_pod.t [hunk 4]
```diff
@@ -32,10 +32,10 @@ subtest 'Can expect module naked sub' => sub {
 subtest 'Test will fail if we increased the number of naked subs' => sub {
     test_out(
         "not ok 1 # TODO & SKIP You have 0.00% POD coverage for the module '$test_module'.",
-        "not ok 2 - Your last changes increased the number of naked subs in the $test_module package.",
+        "not ok 2 - Your last changes increased the number of naked subs in the $test_module package from 2 to 3. Please add sube for your new subs.",
```
Comment (shantanu-deriv) [inline t/no_pod.t]:
```suggestion
        "not ok 2 - Your last changes increased the number of naked subs in the $test_module package from 2 to 3. Please add pod for your new subs.",
```

File: t/no_pod.t [hunk 5]
```diff
@@ -32,10 +32,10 @@ subtest 'Can expect module naked sub' => sub {
...
         "not ok 3 # TODO & SKIP There is no POD in the file $test_module_path."
     );
-    test_diag("  Failed test 'Your last changes increased the number of naked subs in the $test_module package.'",
+    test_diag("  Failed test 'Your last changes increased the number of naked subs in the $test_module package from 2 to 3. Please add sube for your new subs.'",
```
Comment (shantanu-deriv) [inline t/no_pod.t]:
```suggestion
    test_diag("  Failed test 'Your last changes increased the number of naked subs in the $test_module package from 2 to 3. Please add pod for your new subs.'",
```

**Review by shantanu-deriv — CHANGES_REQUESTED** [review]

**Review by shantanu-deriv — APPROVED** [review]

**Pull request merged**